dev-infrastructure/modules/aks-cluster-base.bicep

```diff
@@ -335,6 +337,15 @@ resource aksCluster 'Microsoft.ContainerService/managedClusters@2024-04-02-previ
       ? {
           mode: 'Istio'
           istio: {
+            certificateAuthority: {
+              plugin: {
+                certChainObjectName: 'cert-chain'
+                certObjectName: 'ca-cert'
+                keyObjectName: 'ca-key'
+                keyVaultId: (istioCerticiateKeyVaultID)
+                rootCertObjectName: 'root-cert'
+              }
+            }
```

Comment:
what is the purpose of providing a CA to istio? afaiu this CA is used by istio to create and sign certificates for in-cluster MTLS.